### DISCUSSION TITLE
Question 84: Botany collections website is giving errors. [KH - Botany]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1167] [Q&A] [answered]
Description: Getting a white screen with an error message “no healthy upstream“. Is doing that as of now.

**Comment by rondlg** [comment]
Response: Submit a helpdesk ticket.
- Zoo, Ant and Geo are currently working ok but have intermittently seen it there.

**Comment by rondlg** [comment]
@pdavisfmnh @magpiedin 

I know you worked on this this week. For future reference, if you could drop a note here of what you tried, we'd be grateful. This was also noted by Birds: ZD73622

**Reply by pdavisfmnh** [reply, marked as answer]
The issue is the websites are getting hit by so many different web scraper/indexing bots all at the same time. Essentially causing the site to get a DDOS status and crashing the web server. 

I'm in the process of moving the sites to new infrastructure that should better support the traffic load but also allow blocking of bot traffic we don't want scraping the sites.

**Comment by rondlg** [comment]
From @magpiedin [Related info.](https://www.forbes.com/sites/emmawoollacott/2024/04/16/yes-the-bots-really-are-taking-over-the-internet/)
